# /** ------分配控制器的属性信息 ------ */

string   au1Name
string   au1Value